src/EnergyPlus/HVACUnitaryBypassVAV.cc

```diff
@@ -3714,40 +3588,38 @@ namespace HVACUnitaryBypassVAV {
                 //       Should use CpAirAtCoolSetPoint or CpAirAtHeatSetPoint here?
                 //       If so, use ZoneThermostatSetPointLo(ZoneNum) and ZoneThermostatSetPointHi(ZoneNum)
                 //       along with the zone humidity ratio
-                if (CpSupplyAir * Node(BoxOutletNodeNum).MassFlowRateMax == 0.0) {
-                    SupplyAirTempToHeatSetPt = Node(ZoneNodeNum).Temp;
-                    SupplyAirTempToCoolSetPt = Node(ZoneNodeNum).Temp;
+                if (CpSupplyAir * DataLoopNode::Node(BoxOutletNodeNum).MassFlowRateMax == 0.0) {
+                    SupplyAirTempToHeatSetPt = DataLoopNode::Node(ZoneNodeNum).Temp;
+                    SupplyAirTempToCoolSetPt = DataLoopNode::Node(ZoneNodeNum).Temp;
                 } else {
-                    SupplyAirTempToHeatSetPt = Node(ZoneNodeNum).Temp + QToHeatSetPt / (CpSupplyAir * Node(BoxOutletNodeNum).MassFlowRateMax);
-                    SupplyAirTempToCoolSetPt = Node(ZoneNodeNum).Temp + QToCoolSetPt / (CpSupplyAir * Node(BoxOutletNodeNum).MassFlowRateMax);
+                    SupplyAirTempToHeatSetPt =
+                        DataLoopNode::Node(ZoneNodeNum).Temp + QToHeatSetPt / (CpSupplyAir * DataLoopNode::Node(BoxOutletNodeNum).MassFlowRateMax);
+                    SupplyAirTempToCoolSetPt =
+                        DataLoopNode::Node(ZoneNodeNum).Temp + QToCoolSetPt / (CpSupplyAir * DataLoopNode::Node(BoxOutletNodeNum).MassFlowRateMax);
                 }
                 TSupplyToHeatSetPtMax = max(SupplyAirTempToHeatSetPt, TSupplyToHeatSetPtMax);
                 TSupplyToCoolSetPtMin = min(SupplyAirTempToCoolSetPt, TSupplyToCoolSetPtMin);
             }
         }
 
         //   Account for floating condition where cooling/heating is required to avoid overshooting setpoint
-        if (CBVAV(CBVAVNumber).HeatCoolMode == 0 && CBVAV(CBVAVNumber).OpMode == ContFanCycCoil) {
+        if (CBVAV(CBVAVNumber).HeatCoolMode == 0 && CBVAV(CBVAVNumber).OpMode == DataHVACGlobals::ContFanCycCoil) {
             if (OutAirTemp > TSupplyToCoolSetPtMin) {
                 CalcSetPointTempTarget = TSupplyToCoolSetPtMin;
-                CBVAV(CBVAVNumber).HeatCoolMode = CoolingMode;
             } else if (OutAirTemp < TSupplyToHeatSetPtMax) {
                 CalcSetPointTempTarget = TSupplyToHeatSetPtMax;
-                CBVAV(CBVAVNumber).HeatCoolMode = HeatingMode;
```

Comment:
Lines 3733, 3736, 3747 and 3750 here are responsible for diff's. These lines would change the operating mode based on outdoor temperature and disregard the user specified Priority Control Mode (Cooling Priority, Heating Priority, ZonePriority, and now LoadPriority). Carrier noticed this during the review process for this new feature (See discussion in Conversation tab of 8/9/2019).